package_mrtrix

```diff
@@ -72,7 +64,7 @@ EOF
 
     QTCORE=$(for n in $LIBS; do echo $n; done | grep 'libQt.*Core')
     [[ $QTCORE ]] && ( # sort out Qt:
-      QTPLUGDIR=$(strings $QTCORE | sed -n 's/qt_plugpath=\(.*\)$/\1/p')
+      QTPLUGDIR=/usr/lib/qt/plugins/ #$(strings $QTCORE | sed -n 's/qt_plugpath=\(.*\)$/\1/p')
```

Comment:
Good point. I'd forgotten about this fudge. Problem is that there is no `qt_plugpath` string to be found within my version of the core Qt5 library - my guess they've changed things around, so that approach can't be relied on. This is why I'd hard-coded in the path. I've no idea how to make this robust, other than forcing the user to provide this path as an additional argument/option/environment variable - which isn't great...